src/stores/QueryStore.ts

```diff
@@ -140,7 +156,7 @@ export default class QueryStore extends Store<QueryStoreState> {
                 queryPoints: newPoints,
             }
 
-            return this.routeIfAllPointsSet(newState)
+            return this.routeIfReady(newState)
```

Comment:
I renamed this method because the old name was no longer accurate (because of custom model baut also because of the routing profile before).